pythonforandroid/bootstraps/common/build/templates/build.tmpl.gradle

```diff
@@ -36,6 +36,7 @@ android {
         targetSdkVersion {{ android_api }}
         versionCode {{ args.numeric_version }}
         versionName '{{ args.version }}'
+        manifestPlaceholders = {{ args.manifest_placeholders}}
```

Comment:
I didn't get a chance to check, but I assume this default behaviour you introduced doesn't break anything right?
Could you try to run the auto-compiled test apps on device and confirm it's not breaking?

Reply:
It just adds an empty map, so it shouldn't break anything.
BTW, I just downloaded the artifact for arm64-v8a and it works flawlessly.

Reply:
Nice thanks for giving a try, let's merge :rocket: